frontend/app/admin/src/views/settings/event-types/EditPremiseTypesView.vue

```diff
@@ -4,37 +4,21 @@
             {{ title }}
         </h1>
 
-        <STErrorsDefault :error-box="errors.errorBox" />
-
-        <STList v-model="draggableTypes" :draggable="true">
-            <template #item="{item: type}">
-                <PremiseTypeRow :type="type" @click="editType(type)" />
-            </template>
-        </STList>
-
-        <p>
-            <button class="button text" type="button" @click="addType">
-                <span class="icon add" />
-                <span>{{ $t('8daed6fc-e3dd-4b6b-bd78-c79add8edfb9') }}</span>
-            </button>
-        </p>
+        <PremiseTypesList v-model="draggableTypes" :add-array-patch="addArrayPatch" />
```

Comment:
Probeer niet gerelateerde wijzigingen in een andere commit te zetten in de toekomst